addons/nuxeo-ai-aws-core/src/main/java/org/nuxeo/ai/enrichment/KeyphraseExtractionProvider.java

```diff
@@ -62,6 +67,11 @@ public Collection<EnrichmentMetadata> enrich(BlobTextFromDocument blobTextFromDo
         return AWSHelper.handlingExceptions(() -> {
             List<EnrichmentMetadata> enriched = new ArrayList<>();
             for (Map.Entry<String, String> prop : blobTextFromDoc.getProperties().entrySet()) {
+                if (prop.getValue().length() * 2L > maxSize) {
```

Comment:
I'm not certain if the multiplication is necessary here. The quotas are specifically byte sizes.

Reply:
I tested this and the multiplication here isn't necessary. I've updated the tests to validate this. I also updated the max lengths in a couple files.

I had tested with a large string and got the following error:

```
com.amazonaws.services.comprehend.model.TextSizeLimitExceededException: Input text size exceeds limit. Max length of request text allowed is 100000 bytes while in this request the text size is 999990 bytes (Service: AmazonComprehend; Status Code: 400; Error Code: TextSizeLimitExceededException; Request ID: 688e850c-e9b0-4989-8905-354b22928392; Proxy: null)

	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1879)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleServiceErrorResponse(AmazonHttpClient.java:1418)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1387)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1157)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:814)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:781)
```